agent/config/config.go

```diff
@@ -50,9 +51,12 @@ const (
 	// AgentPrometheusExpositionPort is used to expose Prometheus metrics that can be scraped by a Prometheus server
 	AgentPrometheusExpositionPort = 51680
 
-	// defaultConfigFileName is the default (json-formatted) config file
+	// defaultConfigFileName is the default (json-formatted) config file (non-Windows)
 	defaultConfigFileName = "/etc/ecs_container_agent/config.json"
 
+	// defaultWindowsConfigFileName is the default (json-formatted) config file on Windows
+	defaultWindowsConfigFileName = `C:\ProgramFiles\Amazon\ECS\config.json`
```

Comment:
Should it be ```C:\Program Files\Amazon\ECS\config.json``` (with space in Program Files)?

Reply:
oops good catch, fixed it